packages/bus-query-operation/lib/ActorQueryOperation.ts

```diff
@@ -75,6 +77,25 @@ export abstract class ActorQueryOperation extends Actor<IActionQueryOperation, I
     }
   }
 
+  public static getExpressionContext(context: ActionContext): IExpressionContext {
+    if (context) {
+      const now: Date | undefined = context.get(KEY_CONTEXT_QUERY_TIMESTAMP, undefined);
```

Comment:
Should be equivalent to:
```suggestion
      const now: Date | undefined = context.get(KEY_CONTEXT_QUERY_TIMESTAMP);
```